Model/CategoryData.php

```diff
@@ -12,17 +12,24 @@ public function __construct(
         $this->storeManager = $storeManager;
     }
 
-    public function getCategories($storeId)
+    public function getCategoryQuery($storeId = 0)
```

Comment:
Why have you set this default value here?

Reply:
Agree that it is unnecessary. Removed.